contracts/Permissions.sol

```diff
@@ -334,16 +321,20 @@ library Permissions {
         uint16 role,
         address targetAddress,
         bytes4 functionSig,
+        /*
+         * NOTE: this parameter is not used and as a matter of a fact
+         * it is always invoked from tests with true.
+         *
+         * Even with the current api it has no semantic meaning.
+         * scope false would be achieved by calling setTargetAddressScoped(...,false)
+         *
+         * In a follow up commit will delete. currently keeping the diff minimal
+         */
```

Comment:
@auryn-macmillan now we are ok with the PR, I want to remove the scoped param from this method.